gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Feb 14 14:07:57 MST 2019
+#Wed Feb 24 11:46:10 CET 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
Thanks, totally forgot about wrappers 👍 

I get your point with having `all` over `bin`, I don't have any strong opinion on that especially that size difference is not that big as I thought it can be:
```
$ ls -la gradle-6.1.1-*
.rw-r--r--@ 138M wzieba 26 Feb  8:22 gradle-6.1.1-all.zip
.rw-r--r--@  97M wzieba 26 Feb  8:22 gradle-6.1.1-bin.zip
```
I'll change it to `all`

Reply:
Fixed in #54